extensions/ai-scorer/javascript/ai_scorer.js

```diff
@@ -7,7 +7,6 @@ const handleData = ($component) => {
       const image_data = JSON.parse(json);
 
       image_data.forEach((_data, index) => {
-        galleryData.set(`seed-${_data[`seed-${index}`]}-index-${index}`, _data);
```

Comment:
I forgot to remove this in a previous PR for the scorer extension.  The `galleryData` does not exist anywhere.